packages/parser/package.json

```diff
@@ -80,7 +80,7 @@
     "eslint-plugin-github": "^4.3.7",
     "eslint-plugin-security": "^1.5.0",
     "eslint-plugin-sonarjs": "^0.15.0",
-    "jest": "^29.0.2",
+    "jest": "^29.7.0",
```

Comment:
Please do not upgrade jest if not needed. If you want to update it, please open a separate PR.